### ISSUE TITLE
Archive 2021 EPA-EIA crosswalk outputs

### ISSUE THREAD

**Issue by zschira** [catalyst-cooperative/pudl#2505]
## Background
#2462 demonstrated that it's pretty easy to run EPA-EIA crosswalk with 2021 data. To actually use these results, we need to archive the output in a way that can be easily and consistently accessed.

### Manually generate outputs
The first and simplest step is to just manually run the crosswalk with 2021 data and put the output somewhere. It would probably be easiest to just get these outputs into a git repo to quickly make them accessible.

### Create a reproducible archiving process
While creating a static manually compiled output is a good start, it would probably be good to have a more reproducible programmatic process that will incorporate any data updates, and any updates to the crosswalk repo (this could be process changes or manual mapping additions).

#### Steps to create archiver
- [x] Fork [camd-eia-crosswalk](https://github.com/USEPA/camd-eia-crosswalk)
- [x] Change R-markdown script to use 2021 data
- [x] Remove hardcoded EPA API credentials
- [x] Configure fork to automatically sync with upstream repo
- [x] Add basic R script to execute markdown
- [x] Create [pudl-archiver](https://github.com/catalyst-cooperative/pudl-archiver) that will run crosswalk and archive outputs on zenodo during standard archiving process. Blocked by [issue #90 in the pudl-archiver repo](https://github.com/catalyst-cooperative/pudl-archiver/issues/90). See [issue #102 in the pudl-archiver repo](https://github.com/catalyst-cooperative/pudl-archiver/issues/102)

**Comment by arengel** [comment]
Does the nightly `pudl.sqlite` have the crosswalk created using 2021 data?

**Comment by zschira** [comment]
Rather than dynamically running the crosswalk we decided to just archive a zip of our [fork](https://github.com/catalyst-cooperative/camd-eia-crosswalk-2021) in the same we were previously archiving the 2018 version. This zip contains a static version of the outputs using 2021 data. There still might be a benefit to adding the ability to dynamically generate the outputs as this would allow us to more easily incorporate data updates, and create new versions when there are new data releases, but using this static archive was the quickest way to get the 2021 outputs in use.

@arengel we just merged an update to PUDL that will use the 2021 archive, so, assuming no problems with the build, tomorrows nightly outputs will contain the 2021 crosswalk.
